crates/youki/src/main.rs

```diff
@@ -147,195 +139,208 @@ fn main() -> Result<()> {
     cmd_result
 }
 
-fn determine_root_path(root_path: Option<PathBuf>) -> Result<PathBuf> {
-    let uid = getuid().as_raw();
+mod rootpath {
+    use anyhow::{bail, Result};
+    use libcontainer::rootless::rootless_required;
+    use libcontainer::utils::create_dir_all_with_mode;
+    use nix::libc;
+    use nix::sys::stat::Mode;
+    use nix::unistd::getuid;
+    use std::fs;
+    use std::path::{Path, PathBuf};
+
+    pub fn determine(root_path: Option<PathBuf>) -> Result<PathBuf> {
```

Comment:
It looks like nothing has changed, but is that right?

Reply:
Yes, nothing has changed, except I moved them into a separate mod.